pkg/http/api_test.go

```diff
@@ -4873,6 +4873,171 @@ func TestV1StateEndpoints(t *testing.T) {
 		assert.Equal(t, 500, resp.StatusCode)
 		assert.Equal(t, 2, failingStore.Failure.CallCount("timeoutQueryKey"))
 	})
+
+	t.Run("state save with no content type", func(t *testing.T) {
+		contentTypeStore := &daprt.MockStateStore{}
+		
+		contentTypeStore.On("BulkSet", mock.MatchedBy(matchContextInterface), mock.MatchedBy(func(reqs []state.SetRequest) bool {
+			if (len(reqs) != 1) {
+				return false
+			}
+
+			req := reqs[0]
+
+			if req.ContentType != nil {
+				return false
+			}
+
+			_, ok := req.Metadata["contentType"]
+
+			if ok {
+				return false
+			}
+
+			return true
+		})).Return(nil)
+
+		testAPI.stateStores["contentTypeStore"] = contentTypeStore
+		
+		apiPath := fmt.Sprintf("v1.0/state/%s", "contentTypeStore")
+
+		reqs := []state.SetRequest{
+			{
+				Value: map[string]string{
+					"key": "value",
+				},
+			},
+		}
+		b, _ := json.Marshal(reqs)
+
+		fakeServer.DoRequest("POST", apiPath, b, nil)
+	})
+
+	t.Run("state save with metadata content type", func(t *testing.T) {
+		contentTypeStore := &daprt.MockStateStore{}
+		
+		applicationJson := "application/json"
```

Comment:
This package has a constant `jsonContentTypeHeader` that can be used

Reply:
Used that package's constant instead.